apps/web/components/booking/BookingListItem.tsx

```diff
@@ -413,7 +417,7 @@ function BookingListItem(booking: BookingItemProps) {
               {title}
               <span> </span>
 
-              {!!booking?.eventType?.price && !booking.paid && (
+              {paymentAppData.enabled && !booking.paid && booking.payment.length && (
```

Comment:
Only show the awaiting payment badge if payments are enabled and there was a payment associated with that booking